### DISCUSSION TITLE
Suddenly can't build project

### DISCUSSION THREAD

**Discussion by GamerGirlandCo** [unoplatform/uno#16934] [Q&A] [answered]
```
'MainPage' does not contain a definition for 'Bindings' and no accessible extension method 'Bindings' accepting a first argument of type 'MainPage' could be found (are you missing a using directive or an assembly reference?)
```

i've tried:
- deleting `bin` and `obj`
- recreating the project
- upgrading the uno sdk to a dev version
- cleaning both the solution and the project itself

**Comment by jeromelaban** [comment]
Thanks for the report. Can you provide a [binlog of the build](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)? It is likely that there's a prior error that causes this issue.

**Reply by GamerGirlandCo** [reply]
here are the binlogs [msbuild.zip](https://github.com/user-attachments/files/15507842/msbuild.zip)

**Reply by jeromelaban** [reply, marked as answer]
Thanks! The Binding property is only generated when there are `x:Bind` bindings in the XAML. You can either add one back, or remove the references to `Bindings`, as without `x:Bind` expressions it would not do anything.
